core/src/main/scala/org/apache/spark/internal/config/Tests.scala

```diff
@@ -79,6 +79,7 @@ private[spark] object Tests {
   // on the host.
   val SKIP_VALIDATE_CORES_TESTING =
     ConfigBuilder("spark.testing.skipValidateCores")
+      .version("3.1.0")
```

Comment:
SPARK-29154, commit ID: 474b1bb5c2bce2f83c4dd8e19b9b7c5b3aebd6c4#diff-8b4ea8f3b0cc1e7ce7e943de1abbb165

Reply:
Do we need this? this is an internal testing config, not for users

Reply:
Based on my investigation. In the past, some internal configurations will be changed into public ones.
So it is possible for this configuration to become public in the future.

Reply:
Yeah, we don't really need this strictly.  It's just a good to know when we make it public later. These versions happened to be marked for all other configurations, and these look the only instances left. Should be fine.

Reply:
Thanks.